src/Perspex.SceneGraph/Animation/CrossFade.cs

```diff
@@ -8,12 +8,29 @@
 
 namespace Perspex.Animation
 {
+    // TODO: Perhaps we want a common base for Transitions with Duration so that we can 
+    // consolidate the property declarations, etc
+    //
+
     /// <summary>
     /// Defines a cross-fade animation between two <see cref="IVisual"/>s.
     /// </summary>
-    public class CrossFade : IPageTransition
+    public class CrossFade : PerspexObject, IPageTransition
```

Comment:
I ran into a compile error with the PerspexProperty definition if it was not. Is this ok?